docs/visualize/vega.asciidoc

```diff
@@ -8,6 +8,10 @@ into Kibana, either standalone, or on top of a map.  To see Vega in action,
 watch this
 https://www.youtube.com/watch?v=lQGCipY3th8[short introduction video].
 
+Currently Vega version 4.4 and VegaLite version 2.6 are supported.
+
+WARNING: In Vega it is possible to load data dynamically, e.g. by setting signals as data urls. This is not supported in Kibana as all data is fetched at once prior to passing it to the vega renderer.
```

Comment:
I would call this `Current Limitations`.  "warning" sounds like something is dangerous.

Reply:
You are right, maybe a warning is too much. I don't think Current Limitations is supported, I changed it to `NOTE: `. It should be rendered like this:
<img width="797" alt="screenshot 2019-02-12 at 17 00 13" src="https://user-images.githubusercontent.com/1508364/52649009-c7727180-2ee7-11e9-94d3-e5c6b5a68404.png">